web/app/js/components/StatusTable.jsx

```diff
@@ -36,11 +53,7 @@ const columns = {
       render: statuses => {
         return _.map(statuses, status => {
           // TODO: handle case where there are too many dots for column
-          return <div
-            className={`status-dot status-dot-${status.value}`}
-            key={status.name}
-            title={status.name}
-          >&nbsp;</div>
+          return <StatusDot status={status} />
```

Comment:
I think this component needs a `key` prop that's unique. Otherwise it will warn to the console.

Reply:
ah, good catch!